cmd/mms/commands.go

```diff
@@ -138,3 +146,39 @@ func productReceiver(event *mms.ProductEvent) error {
 	fmt.Println(event)
 	return nil
 }
+
+func createExecutableCallback(filepath string, filter bool) func(event *mms.ProductEvent) error {
+	_, err := exec.LookPath(filepath)
+	fmt.Println(filepath)
+	if err != nil {
+		panic(fmt.Errorf("command executable not found, %s", err.Error()))
+
+	}
+
+	return func(event *mms.ProductEvent) error {
+		var productslug string
+
+		if filter {
+			productslug = event.Product
```

Comment:
What if we rather gave the user the ProductLocation? That could be directly used to process it. The Product name is less useful in that respect. Eventually we should provide more of those fields to the user, though right now the ProductLocation might make the most sense?

Reply:
Fixed by caf2fed